src/TenorGuitar/FingerStatus.purs

```diff
@@ -0,0 +1,47 @@
+module TenorGuitar.FingerStatus (FingerStatus(..)) where
```

Comment:
I don't think this file is necessary is it?  It seems to have found its way in from ```Bass``` where it's needed because you can have two finger positions on a single string. And as far as I can tell it's not called anywhere.

Reply:
Hi, I deleted this file and pushed an update. Yes, you are correct. I must have accidentally left it in. I actually first started by using the bass as a template but then realized that the Guitar was the better template as you had also pointed out in a message.